app/controllers/tickets_controller.rb

```diff
@@ -1,2 +1,49 @@
 class TicketsController < ApplicationController
+  before_action :find_ticket, only: %i[show edit update destroy]
+
+  def index
+    @tickets = Ticket.all
+  end
+
+  def show
+    @vote = Vote.new
+    @comment = Comment.new
+  end
+
+  def new
+    @ticket = Ticket.new
+  end
+
+  def create
+    @ticket = Ticket.create(ticket_params)
+    @ticket.user_id = current_user.id
+    redirect_to ticket_path(@ticket)
```

Comment:
Salut Julien,
Il y a un soucis sur la ligne 20 et aussi sur le loup.
Je pense que tu a oublié de changer la syntax, et aussi je ne sais pas si tu veux retourner sur le ticket_path avant de se assurer que le ticket a était créé.
Merci 😃

Reply:
The rest looks good!! 🙏🏼

Reply:
Bien vu! Je vire la ligne 20, la boucle me semble bonne par contre, si le ticket est enregistré, cela nous renvoie vers la show du ticket